packages/x-charts/src/ChartsTooltip/useAxisTooltip.tsx

```diff
@@ -22,10 +23,24 @@ export interface UseAxisTooltipReturnValue<
   AxisValueT extends string | number | Date = string | number | Date,
 > {
   axisDirection: 'x' | 'y';
-  dataIndex: number;
-  seriesItems: SeriesItem<SeriesT>[];
+  axisId: AxisId;
   axisValue: AxisValueT;
   axisFormattedValue: string;
+  dataIndex: number;
+  seriesItems: SeriesItem<SeriesT>[];
+}
+
+export interface UseAxisTooltipParams {
+  /**
+   * If `true`, the hook returns an array with an object per active axis.
+   * @deprecated In future version, the `multipleAxes=true` will be the unique behavior
+   */
+  multipleAxes?: boolean;
```

Comment:
Instead of overloading `useAxisTooltip` would it make sense to create a `useAxesTooltips` instead and deprecate this `useAxisTooltip`? 

I suppose it would make it easier to identify the deprecated code, as well as writing a codemod. Plus, the hook name would more closely follow what's happening: we're obtaining the tooltip content of all axes.

Reply:
I tried to do it without breaking changes, And then realised I had to modify the tooltip content.

Based on your comment, I created `useAxesTooltips(params) = useAxisTooltips({...params, multipleAxies: true})` to avoid having similar logic in tow files. In next major we will be able to merge the two file together.